examples/content_management_examples/common/examples/todos/todos.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema } from '@kbn/config-schema';
+import {
+  CreateIn,
+  DeleteIn,
+  GetIn,
+  SearchIn,
+  UpdateIn,
+} from '@kbn/content-management-plugin/common';
+
+export const TODO_CONTENT_ID = 'todos';
+export interface Todo {
+  id: string;
+  title: string;
+  completed: boolean;
+}
+const todoSchema = schema.object({
+  id: schema.string(),
+  title: schema.string(),
+  completed: schema.boolean(),
+});
+
+export type TodoCreateIn = CreateIn<'todos', { title: string }>;
+export type TodoCreateOut = Todo; // TODO: Is this correct?
+export const createInSchema = schema.object({ title: schema.string() });
+export const createOutSchema = todoSchema;
+
+export type TodoUpdateIn = UpdateIn<'todos', Partial<Omit<Todo, 'id'>>>;
+export type TodoUpdateOut = Todo;
+export const updateInSchema = schema.object({
+  title: schema.maybe(schema.string()),
+  completed: schema.maybe(schema.boolean()),
+});
+export const updateOutSchema = todoSchema;
+
+export type TodoDeleteIn = DeleteIn<'todos', { id: string }>;
+export type TodoDeleteOut = void;
+// no schema for delete ?
+
+export type TodoGetIn = GetIn<'todos'>;
+export type TodoGetOut = Todo;
+// no get in schema for get ?
+export const getOutSchema = todoSchema;
+
+export type TodoSearchIn = SearchIn<'todos', { filter?: 'todo' | 'completed' }>;
+export interface TodoSearchOut {
+  hits: Todo[];
+}
+export const searchInSchema = schema.object({
```

Comment:
This should be a `query` schema to map to that object. Not "searchInSchema".  And it shoud be declared server side.